front/app/containers/Admin/reporting/containers/FullScreenReport/index.tsx

```diff
@@ -25,11 +25,11 @@ const Centerer = styled.div`
 
   @media print {
     display: block;
-  }
-
-  @page {
-    size: auto;
-    margin: 0mm;
+    position: absolute;
+    @page {
+      size: auto;
+      margin: 30px 0;
```

Comment:
This was Fran's simple fix! Had tried all types of combination of margin-top margin-bottom etc, but this appears to fix the issue